ospfd/ospf_vty.c

```diff
@@ -3365,6 +3365,56 @@ DEFUN (show_ip_ospf_instance,
 	return ret;
 }
 
+static void ospf_interface_auth_show(struct vty *vty, struct ospf_interface *oi,
+				     json_object *json, bool use_json)
+{
+	int auth_type;
+
+	auth_type = OSPF_IF_PARAM(oi, auth_type);
+
+	switch (auth_type) {
+	case OSPF_AUTH_NULL:
+		if (use_json)
+			json_object_string_add(json, "authType",
+					       "NULL Authentication");
```

Comment:
you've used json info - like key strings - that differ from what's used in `show_ip_ospf_area()` - was there a reason not to be consistent with that output?

Reply:
To be consistent b/w interface and area level authentication information , Json strings modified as like area level json strings.

Reply:
It still looks to me as if the json keys in `show_ip_ospf_area()` are different from this PR?

Reply:
As suggested , json keys also changed as like area level auth o/p.